Zend/zend_object_handlers.c

```diff
@@ -62,7 +62,7 @@
   called, we cal __call handler.
 */
 
-ZEND_API void rebuild_object_properties(zend_object *zobj) /* {{{ */
+static void rebuild_object_properties(zend_object *zobj) /* {{{ */
```

Comment:
since something no-longer is `ZEND_API` which was before, does this need to be noted in some changelog/releasenotes ?

Reply:
Agreed. I usually update the UPGRADING files separately, as our merge strategy for those tends to lose the changes.